saul-core/src/main/scala/edu/illinois/cs/cogcomp/saul/datamodel/node/InstanceSet.scala

```diff
@@ -54,18 +52,3 @@ case class NodeSet[T <: AnyRef](node: Node[T]) extends InstanceSet[T] {
 case class SingletonSet[T <: AnyRef](node: Node[T], t: T) extends InstanceSet[T] {
   val instances: Iterable[T] = Set(t)
 }
-
-trait PropertySet[T <: AnyRef, V] extends Iterable[V] {
-  self =>
-  def property: TypedProperty[T, V]
-  def underlying: InstanceSet[T]
-  lazy val propValues: Iterable[V] = {
-    val ab = new ArrayBuffer[V]
-    ab ++= underlying.map(property(_))
-    ab
-  }
-
-  override def iterator: Iterator[V] = propValues.iterator
-
-  def counts = propValues.groupBy(x => x).map(p => p._1 -> p._2.size).toMap
-}
```

Comment:
Moved `PropertySet` to a separate Scala file. Seemed different from `InstanceSet`